components/popover/__docs__/storybook-stories.jsx

```diff
@@ -123,4 +125,8 @@ storiesOf(POPOVER, module)
 			containerStyle: { background: containerBackgroundColor },
 			style: { background: popoverBackgroundColor },
 		})
-	);
+	)
+	.add('Error', () => <Error />)
```

Comment:
Is it necessary to have both open and close?

Reply:
One is for the visual regression tests and the other is to open and close it. I guess we could just have the open one since the logic is the same as the base Popover.